qiskit/quantum_info/operators/channel/kraus.py

```diff
@@ -69,23 +69,17 @@ def __init__(
         """Initialize a quantum channel Kraus operator.
 
         Args:
-            data (QuantumCircuit or
-                  Instruction or
-                  BaseOperator or
-                  matrix): data to initialize superoperator.
-            input_dims (tuple): the input subsystem dimensions.
-                                [Default: None]
-            output_dims (tuple): the output subsystem dimensions.
-                                 [Default: None]
+            data: data to initialize superoperator.
+            input_dims: the input subsystem dimensions. (Default: `None`)
+            output_dims: the output subsystem dimensions. (Default: `None`)
```

Comment:
I don't think you need to write the defaults since they're given in the signature, but if you do want them there as well, you probably meant ` ``None`` ` for each.

Reply:
good point. Removing them in 8f2bc891d0826d933e2a3b104e98afb5fe2998b5